src/lib/embed.js

```diff
@@ -67,6 +67,7 @@ export function createEmbed({ html }, element) {
     const div = document.createElement('div');
     div.innerHTML = html;
 
+    div.firstChild.setAttribute('data-vimeo-wrapper', 'true');
```

Comment:
would it be possible to not add this data attribute? to not add too much to the DOM